tutorials/deploy-edge/production-deploy-to-ase-gpu.ipynb

```diff
@@ -0,0 +1,449 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploying a ML model as web service on Azure Stack\n",
+    "This notebook shows the steps to : registering a model, creating an image, provisioning,deploying a service using Iot Edge on Azure Edge. ![aml-flow](img/ml-flow-edge.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import azureml.core\n",
+    "\n",
+    "print(azureml.core.VERSION)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Get workspace\n",
+    "Please create a azure Machine learnign workspace on portal.azure.com before runing this notebook, once created download config.json from your workspace,  please place config.json file from portal to same folder as notebook ![Capture_withoverlay.JPG](img/conf_file_download.JPG)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.workspace import Workspace\n",
+    "\n",
+    "ws = Workspace.from_config()\n",
+    "print(ws.name, ws.resource_group, ws.location, ws.subscription_id, sep=\"\\n\")"
```

Comment:
print isn't needed, just end the cell with `ws` and it'll print it out

Reply:
It sure will, but it is easier to have it like this so it could be cut/paste into a stand-alone Python script. And it is a convenient placeholder for a more elaborate ws dump with help to the user how to retrieve the resource group/subscription id/etc. Do you mind if I'll leave it as is?

Reply:
Leaving the print - it has value in the context of this notebook, it demoes how to extract information from the workspace object.